terraform/modules/monitoring/alert.tf

```diff
@@ -0,0 +1,132 @@
+resource "google_monitoring_notification_channel" "basic" {
+  project = var.project_id
+
+  display_name = "owner notification channel"
+
+  type = "email"
+
+  labels = {
+    email_address = var.notification_channel_email
+  }
+
+  force_delete = false
+}
+
+resource "google_monitoring_alert_policy" "public_key_service_alert" {
+  project = var.project_id
+
+  display_name = "Public-Key Service Alert: Too many none-200 response"
+
+  combiner = "AND"
+
+  conditions {
+    display_name = "Too many None-200 response"
+
+    condition_threshold {
+      filter          = "metric.type=\"run.googleapis.com/request_count\" resource.type=\"cloud_run_revision\" resource.label.\"service_name\"=\"${var.public_key_service_name}\" AND metric.label.\"response_code\"!=\"200\""
+      duration        = "0s"
+      comparison      = "COMPARISON_GT"
+      threshold_value = 5
+
+      aggregations {
+        alignment_period   = "60s"
+        per_series_aligner = "ALIGN_MEAN"
+      }
+    }
+  }
+
+  notification_channels = [
+    resource.google_monitoring_notification_channel.basic.name
+  ]
+
+  alert_strategy {
+    auto_close = "604800s"
```

Comment:
Should we have auto_close?

Reply:
If we want to setup alert notification, the auto_close field is required.
https://screenshot.googleplex.com/7FLxQJifLS3h2E2

Reply:
Terraform showes it's optional https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/monitoring_alert_policy#nested_alert_strategy
I thought to set up alert notfication, we just need to provide the notification channel?

Reply:
Just tried in terraform. Removed `auto_close`